scripts/prepare_release.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash -
+# Usage: ./scripts/prepare_release.sh <extension project> <release version>
+# Example: ./scripts/prepare_release.sh jib-layer-filter-extension-gradle 0.2.0
+
+set -o errexit
+
+EchoRed() {
+  echo "$(tput setaf 1; tput bold)$1$(tput sgr0)"
+}
+EchoGreen() {
+  echo "$(tput setaf 2; tput bold)$1$(tput sgr0)"
+}
+
+Die() {
+  EchoRed "$1"
+  exit 1
+}
+
+DieUsage() {
+  EchoRed "Usage: ./scripts/prepare_release.sh <extension project> <release version> [<post-release-version>]"
+  EchoRed
+  EchoRed "Example: ./scripts/prepare_release.sh jib-layer-filter-extension-gradle 0.2.0"
```

Comment:
Does this need to be `/` instead of `-` in `<extension project>` value in the example
```diff
-   EchoRed "Example: ./scripts/prepare_release.sh jib-layer-filter-extension-gradle 0.2.0"
+   EchoRed "Example: ./scripts/prepare_release.sh jib-layer-filter-extension/gradle 0.2.0"
```

Reply:
The name comes from [`settings.gradle`](https://github.com/GoogleContainerTools/jib-extensions/blob/master/settings.gradle) and independent of file path. It's a Gradle module name you will use like `./gradlew :jib-layer-filter-extension-gradle`. And then for simplicity, I'm just reusing that module name when generating tag and branch names. We can change each module name to whatever we want in `settings.gradle`. Maybe `jib-layer-filter-extension-gradle` is too long and `layer-filter-gradle` is good enough? Should we still change the directory structure?

Reply:
Let's move forward with this. I'll reorg the directories (removing `maven/` and `gradle/` directories and so on) after all open PRs are merged.